### PR TITLE
breaking Daves code

### PR DESCRIPTION
**Pull request by chrisgorgo**

### CODE REVIEW

**Comment by nicholsn** [comment]
Hey this breaks the code @chrisfilo !

**Comment by jbpoline** [comment]
That's a crazy request !!!

File: use-cases/abide/ABIDE_to_NIDM_RDFLib.ipynb [hunk 1]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#Import Python libraries\n",
```
Comment (nicholsn) [inline use-cases/abide/ABIDE_to_NIDM_RDFLib.ipynb:7]: Here is where it breaks!

File: use-cases/abide/abide_terms.ttl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 @prefix nidash: <http://purl.org/nidash/nidm/> .
 @prefix nidm: <http://nidm.nidash.org/> .
 @prefix prov: <http://www.w3.org/ns/prov#> .
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdf: <http://www.w3.org/1999/02/21-rdf-syntax-ns#> .
```
Comment (chrisgorgo) [inline use-cases/abide/abide_terms.ttl:6]: I don't think this is right!

**Comment by dbkeator** [comment]
Hey this is wrong!

**Pull request closed without merging**